### PR TITLE
Modified docs in Quaternion

### PR DESCRIPTION
**Pull request by evbernardes**
#### Brief description of what is fixed or changed
Minor edit to some of the method descriptions in `algebras.Quaternion`, as they are a bit hard to follow.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by sylee957** [comment]
It looks like there's a problem with test migrations in CI.
So I may defer the merges before the issues are fixed.

**Comment by evbernardes** [comment]
> It looks like there's a problem with test migrations in CI. So I may defer the merges before the issues are fixed.

I see, so it's an issue elsewhere, non related to the PR?
